tensorflow/core/kernels/mkl/mkl_quantized_pooling_ops_test.cc

```diff
@@ -109,43 +116,54 @@ TEST_F(QuantizedPoolingTest, SmallAveragePooling) {
                             input_quantized.flat<quint8>());
   AddInputFromArray<float>(TensorShape({1}), {input_min});
   AddInputFromArray<float>(TensorShape({1}), {input_max});
-  AddInputFromArray<uint8>(dummy_shape, dummy_tensor);
-  AddInputFromArray<uint8>(dummy_shape, dummy_tensor);
-  AddInputFromArray<uint8>(dummy_shape, dummy_tensor);
+  if (!NativeFormatEnabled()) {
+    AddInputFromArray<uint8>(dummy_shape, dummy_tensor);
+    AddInputFromArray<uint8>(dummy_shape, dummy_tensor);
+    AddInputFromArray<uint8>(dummy_shape, dummy_tensor);
+  }
 
   TF_ASSERT_OK(RunOpKernel());
 
   const Tensor& output = *GetOutput(0);
-  const Tensor& mkl_shape_tensor = *GetOutput(3);
-  ConvMklToTF conv_comp;
   Tensor output_quantized;
-  conv_comp.ConvertMKL2TF<quint8>(DT_QUINT8, output, mkl_shape_tensor,
-                                  output_quantized);
+  if (!NativeFormatEnabled()) {
```

Comment:
This part can use the refactored function [mentioned in #45108](https://github.com/tensorflow/tensorflow/pull/45108#discussion_r549582571).

Reply:
I used the refactored function form #45108.